config.json

```diff
@@ -76,7 +76,8 @@
     "connect",
     "ledger",
     "poker",
-    "variable-length-quantity"
+    "variable-length-quantity",
+    "diamond"
```

Comment:
any thoughts on moving this up earlier to get the exposure a bit earlier? I'm only blindly guessing at the difficulty, but it seems once someone understands how to work with the tests it the task itself can be less complex than, say, `connect` and `poker`. What about moving it before, say, `react`?

We don't have to worry too much about getting the order exactly right though - we'll have a better idea of the order when #345 is complete. But let's try to make a guess now and we can adjustg later